x-pack/plugins/index_management/__jest__/client_integration/home/data_streams_tab.test.ts

```diff
@@ -238,7 +238,7 @@ describe('Data Streams tab', () => {
       const { table, actions } = testBed;
       await actions.clickIndicesAt(0);
       expect(table.getMetaData('indexTable').tableCellsValues).toEqual([
-        ['', '', '', '', '', '', '', 'dataStream1'],
+        ['', 'data-stream-index', '', '', '', '', '', '', 'dataStream1'],
```

Comment:
@sebelga Sorry to ask you a question that I could answer myself, but I'm trying to wrap up a lot of stuff this week... in the PR description you mention that Dashboard, Visualization, Maps, and Graphs are affected by this change. If that's so, then why is this test changing? Are there other UIs that are also being affected, which should be tested? Thanks!

Reply:
It was a fix I did in the TestBed, which in turn fixed this test that had a wrong expectation.

See my comment https://github.com/elastic/kibana/pull/138244/files#r940229580 (and your response 😄 )

Reply:
> Are there other UIs that are also being affected, which should be tested? Thanks!

No this was really just about the Testbed helper to retrieve an EUI table content. The logic changed (to actually retrieve correctly the complete table content) and just one test got affected. Other tables are probably not using the `th` tag.